tasks/eslint.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('eslint', {
+    app: {
+      eslintrc: '.eslintrc',
+      src: ['Gruntfile.js', 'tasks/*.js', 'src/*.js', 'scripts/*.js', 'lib/*.js']
```

Comment:
@TDA, 

1. ESLint should be smart enough to not lint ./node_modules/ by default.
1. Re: `!examples/**/*`, this is probably what .eslintignore is for. Just add "examples" in that file and it will recursively ignore everything. We could also just inline disable the `no-console` rule, if that is the only violator.
1. Re: `!test/**/*`, I guess you maybe have to leave that as an exclude since you explicitly have it as a target below on line 13.

I'd think that since everything uses the same global .eslintrc, probably just easiest to have one target and merge everything together.

Just my $0.01 CAD.